targets/win32/host_linux

```diff
@@ -38,6 +38,11 @@
 win_sysroot=`ls -1d ${WIN32SDK}/i?86-*mingw32* | head -n 1`
 win_cross=`ls -1 ${WIN32SDK}/bin/*mingw32*-gcc | head -n 1 | sed 's/gcc$//'`
 
+test -f ${SYS_PREFIX}/include/WinSock2.h || ln -s $win_sysroot/include/winsock2.h ${SYS_PREFIX}/include/WinSock2.h
```

Comment:
Why are you doing this instead of adding another `-I$win_sysroot/include` in targets/win32/build-binary#L89? Also, does this filename case matter?